Libraries/PeriphDrivers/Source/RTC/rtc_me30.c

```diff
@@ -65,6 +65,30 @@ int MXC_RTC_SetSubsecondAlarm(uint32_t rssa)
     return MXC_RTC_RevA_SetSubsecondAlarm((mxc_rtc_reva_regs_t *)MXC_RTC, rssa);
 }
 
+int MXC_RTC_SetClockSource(mxc_rtc_clock_t clk_src)
+{
+    uint8_t retval = E_NO_ERROR;
+
+    switch (clk_src) {
+    case MXC_RTC_ERTCO_CLK:
+        MXC_GCR->clkctrl |= MXC_F_GCR_CLKCTRL_ERTCO_EN;
+        MXC_MCR->ctrl &= ~MXC_F_MCR_CTRL_CLKSEL;
+        MXC_MCR->ctrl |= MXC_S_MCR_CTRL_CLKSEL_ERTCO;
```

Comment:
```suggestion
        MXC_SETFIELD(MXC_MCR->ctrl, MXC_F_MCR_CTRL_CLKSEL, MXC_S_MCR_CTRL_CLKSEL_ERTCO);
```
I think you should use MXC_SETFIELD macro here.